deploy/src/main/resources/quick-start/README.md

```diff
@@ -0,0 +1,214 @@
+      </tr>
+	  <tr>
+        <td>CAF_RABBITMQ_PASSWORD</td>
+        <td>guest</td>
+        <td>Used to specify the password used to connect to RabbitMQ.</td>
+      </tr>
+	  <tr>
+        <td>CAF_AUTOSCALER_MAXIMUM_INSTANCES</td>
+        <td>100</td>
+        <td>Used to specify the maximum number of instances that any worker can be scaled to.</td>
+      </tr>
+	  <tr>
+		<td>CAF_AUTOSCALER_DOCKER_SWARM_STACK</td>
+		<td>unspecified</td>
+		<td>Used to identify the stack within which the Autoscaler will monitor</td>
+	  <tr>
+        <td>CAF_DOCKER_SWARM_TIMEOUT</td>
+        <td>30</td>
+        <td>Used to specify the max length of time in seconds that a docker REST call can take before a timeout out occurs.</td>
+      </tr>
+	  <tr>
+        <td>CAF_DOCKER_SWARM_HEALTHCHECK_TIMEOUT</td>
+        <td>5</td>
+        <td>Used to specify the max length of time in seconds that the Docker endpoint healthcheck can take before a timeout occurs.</td>
+      </tr>
+    </table> 
+
+3. **Set the service labels**
+
+	For each of the services to scale its docker-compose template must be updated to include labels such as the following example:
+
+	```
+   		labels:
+      		- autoscale.metric=rabbitmq
+      		- autoscale.scalingtarget=autoscale-monitor-in
+      		- autoscale.scalingprofile=default
+      		- autoscale.maxinstances=4
+      		- autoscale.mininstances=1
+      		- autoscale.interval=30
+      		- autoscale.backoff=10
+	```
+
+	The `autoscale.metric` should always be `rabbitmq` for this container. Set the `autoscale.interval` to something sensible given your service workload. If you set this to a very small value (very frequent) then you will get very unstable scaling - the number of instances will rapidly increase and decrease seemingly with no cause. if you set this very large (infrequent) then the scaling will lag behind the demand significantly. Thus, containers which rapidly consume messages can have a lower interval. For containers that consume messages very slowly (whereby the average is tens of minutes or hours) this algorithm may not be suitable for your workloads and you may wish to consider writing a `workload` module more suited to your needs and make a container with it in.
+
+	The `autoscale.scalingtarget` should be the RabbitMQ queue name to monitor. Set `autoscale.maxinstances` and `autoscale.mininstances` as your quality of service and resources dictate. The `autoscale.backoff` is the number of intervals to skip monitoring after a scale up or down command is issued. This prevents unusual values being considered when the system is in an unstable state.
+
+	Deploy/redeploy the services and the autoscale container. After one or two minutes the autoscale container should find the services and start monitoring.
+
+	##### Example
+
+	The Job Service microservices already have these labels specified. You can see the labels set under some of the components, for example with worker-langdetect.
```

Comment:
Dont mention jobservice microservices, as they dont have these labels, ( you are deleting them from its deploy file ).
We do have example workers which are being scaled in our autoscalerdemo, which you should reference instead.